mypy.ini

```diff
@@ -304,6 +304,9 @@ ignore_errors = True
 [mypy-scipy.sparse.tests.test_base]
 ignore_errors = True
 
+[mypy-scipy.special._precompute.cosine_cdf]
```

Comment:
This is yet a third way :) I missed this PR while I was working on another, see e.g. 
https://github.com/scipy/scipy/pull/13721
It seemed to pass on my branch.

Reply:
This is also OK to do considering there are no stubs implemented for this file. But I am in favor of #13721 as it just addresses the errors we see on master instead of ignoring the entire file.